apps/ios/Shared/Views/NewChat/AddGroupView.swift

```diff
@@ -204,7 +206,7 @@ struct AddGroupView: View {
                 chat = c
             }
         } catch {
-            dismiss()
+            dismissAllSheets()
```

Comment:
is there a reason to dismiss all in case of error? why not go back / allow to go back? maybe it's ok, just wondering